libs/settings.js

```diff
@@ -1221,13 +1221,14 @@ settings.panels = {
 			{
 			"numb": "",
                         "isnew": "false",
-                        "highlight": "", // danger = red; success = green;
-			"name": "exvo",
-			"cost": "0,25 BTC ",
-			"status": "Need funds",
+                        "highlight": "danger", // danger = red; success = green;
+			"name": "Exvo",
+			"cost": "1000 EXVO Approximately 110.000 BTCZ ",
```

Comment:
Should probably be `110,000`

Reply:
The result of the vote is 34 yes and 4 no, I think we should contribute the rest of the funds of the community and end this issue already to be able to continue with new proposals.